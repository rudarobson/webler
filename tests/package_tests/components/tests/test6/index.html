<!DOCTYPE html>
<html>
<!--components:
	import simples/simple-component;
	import simples/simple-component as simple-component-as-other-name;
	import simples/simple-component-2 as simple-2;
-->
<body>
	<simple-component class="substitute-blah" merged-empty-attribute merged-nonempty-attribute="a value"></simple-component>
	<simple-2></simple-2>
	<simple-component-as-other-name></simple-component-as-other-name>
</body>

</html>
