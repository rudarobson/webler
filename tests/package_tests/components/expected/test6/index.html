<!DOCTYPE html><html><body><div class="simple-component substitute-blah" merged-empty-attribute merged-nonempty-attribute="a value"></div><div class="simple-component-2"></div><div class="simple-component"></div></body></html>